backend/persistent/datastores/Datastores.js

```diff
@@ -5,7 +5,9 @@ import * as ApacheDrill from './ApacheDrill';
 import * as IbmDb2 from './ibmdb2';
 import * as ApacheLivy from './livy';
 import * as ApacheImpala from './impala';
+import * as DataStoreMock from './DataStoreMock';
```

Comment:
`%s/DataStore/datastore/g`

Reply:
use lower case filenames (otherwise, when developing in mac, it's very easy to commit filename typos)